src/app/routes/article/article.service.ts

```diff
@@ -235,10 +240,20 @@ export const createArticle = async (article: any, id: number) => {
     },
   });
 
-  return articleMapper(createdArticle, id);
+  const cacheKey = `article:${slug}`;
```

Comment:
Probably not worth caching right after creating - read through cache in query endpoints should be enough.